### PR TITLE
Id form unit test

### PR DESCRIPTION
**Pull request by timkeresey**
### What does this PR do?

- begins the testing block for App.
- writes a unit test for the rendering of IDForm.

### Where should the reviewer start?

- IDForm.test.js

### How should this be manually tested?

- By running the testing suite in the terminal

### CODE REVIEW

File: src/components/App/App.test.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
+import { BrowserRouter } from 'react-router-dom';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+describe('App', () => {
```
Comment (taylorjohnson141) [inline src/components/App/App.test.js:5]: Thank you for doing this, I wasn't sure if we should just remove the app tests or not

File: src/components/IDForm/IDForm.test.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import IDForm from './IDForm.js';
+import { render, screen } from '@testing-library/react';
+
+
+describe('IDForm', () => {
```
Comment (taylorjohnson141) [inline src/components/IDForm/IDForm.test.js:7]: Nice job with this, I wasn't sure if we should test the whole thing or only one piece to begin with. If we test that one piece shows up then we can imply it all shows up I think. However it feels safer testing that all the pieces of the component render!

**Review by taylorjohnson141 — APPROVED** [review]
Looks great!

File: src/components/IDForm/IDForm.test.js (hunk 2 above)
Comment (ericberglund117) [reply src/components/IDForm/IDForm.test.js:7]: I agree with taylor. It's like with Unit testing and integration testing.

**Pull request merged**